### PR TITLE
💻 Add types and mock data for events and clubs

### PR DESCRIPTION
**Pull request by ShreyasPrasad**
## Purpose
Added mock data for frontend dev.

### CODE REVIEW

**Review by jeffzh4ng — DISMISSED** [review]

File: client/context/Base.tsx [hunk 1]
```diff
@@ -12,47 +12,107 @@ export type Club = {
...
+  backgroundImageURL?: string
+  tags: Array<string>
 }
 
 export type Event = {
   id: Id
-  name: string
-  description: string
+  name: string,
+  club: string,
```
Comment (xujustinj) [inline client/context/Base.tsx:36]: should be a `BasicClub` based on `https://github.com/loolabs/waterpark/pull/71/files#diff-469d1105c94fbfbe41d10aa21f68b1de04725545cf788b8ca5b18ea6afbd0395R3-R16`

File: client/context/Base.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { createContext, useContext, useEffect, useState } from 'react'
 import { indexData } from '../utils'
+import moment from 'moment'
```
Comment (xujustinj) [inline client/context/Base.tsx:3]:
```suggestion
import moment, { Moment } from 'moment'
```
does this work?

File: client/context/Base.tsx [hunk 3]
```diff
@@ -12,47 +13,107 @@ export type Club = {
...
 export type Event = {
   id: Id
-  name: string
-  description: string
+  name: string,
+  club: string,
+  description: string,
+  backgroundImageURL: string,
+  startDate: moment.Moment,
+  endDate: moment.Moment,
```
Comment (xujustinj) [inline client/context/Base.tsx:41]:
```suggestion
  startDate: Moment,
  endDate: Moment,
```

**Review by xujustinj — DISMISSED** [review]
lgtm 🚀

**Review by jeffzh4ng — DISMISSED** [review]

**Review by xujustinj — APPROVED** [review]

**Review by xujustinj — APPROVED** [review]

**Review by jeffzh4ng — APPROVED** [review]

**Pull request merged**